src/test/kotlin/it/unibo/protelis/web/execution/ProtelisNodeTest.kt

```diff
@@ -0,0 +1,49 @@
+package it.unibo.protelis.web.execution
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.vertx.core.json.Json
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertNotNull
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+class ProtelisNodeTest {
+
+  @Test
+  @DisplayName("Test ProtelisNode comparisons")
+  fun `test ProtelisNode`() {
+    val id = "Pippo"
+    val coordinates = 0 to 1
+    val n = ProtelisNode(id, coordinates)
+    assertNotNull(n)
+    assertEquals(ProtelisNode(id, coordinates), n)
+    assertNotEquals(ProtelisNode("Pluto", coordinates), n)
+    assertNotEquals(ProtelisNode(id, 0 to 0), n)
+    assertEquals(ProtelisNode(id, coordinates).hashCode(), n.hashCode())
+  }
+
+  @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicFunction - [test JSON serialization] at /src/src/test/kotlin/it/unibo/protelis/web/execution/ProtelisNodeTest.kt:26:3](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)